Japx/Classes/Core/Japx.swift

```diff
@@ -27,6 +27,63 @@ public enum JapxError: Error {
     case unableToConvertDataToJson(data: Any)
 }
 
+/// `JapxDecodingOptions` is a set of options affecting the decoding of JSON:API into JSON you requested from `Japx.Decoder`.
+public struct JapxDecodingOptions {
```

Comment:
Shouldn't we move this to `Japx.Decoder` namespace?

Reply:
Moved to `Japx.Decoder.Options`